miniapps/meshing/mesh-optimizer.cpp

```diff
@@ -155,26 +145,47 @@ double ind_values(const Vector &x)
       Ymod= -X*std::sin(thval) + Y*std::cos(thval);
       X = Xmod+0.5; Y = Ymod+0.5;
       double r1 = 0.45; double r2 = 0.55; double sf=30.0;
-      val = ( 0.5*(1+std::tanh(sf*(X-r1))) - 0.5*(1+std::tanh(sf*(X-r2)))
-              + 0.5*(1+std::tanh(sf*(Y-r1))) - 0.5*(1+std::tanh(sf*(Y-r2))) );
-      if (rval > 0.4) {val = 0.;}
-      if (val > 1.0) {val = 1.;}
-      if (val < 0.0) {val = 0.;}
-
-      return val * small + (1.0 - val) * big;
+      val = (0.5*(1+std::tanh(sf*(X-r1))) - 0.5*(1+std::tanh(sf*(X-r2))) +
+             0.5*(1+std::tanh(sf*(Y-r1))) - 0.5*(1+std::tanh(sf*(Y-r2))));
+      if (rval > 0.4) { val = 0.; }
    }
-
-   if (opt==6)
+   else if (opt == 6)
    {
-      double val = 0.;
       const double xc = x(0) - 0.0, yc = x(1) - 0.5;
       const double r = sqrt(xc*xc + yc*yc);
       double r1 = 0.45; double r2 = 0.55; double sf=30.0;
       val = 0.5*(1+std::tanh(sf*(r-r1))) - 0.5*(1+std::tanh(sf*(r-r2)));
-      if (val > 1.) {val = 1;}
-      if (val < 0.) {val = 0;}
+   }
+
+   val = std::max(0.,val);
+   val = std::min(1.,val);
+
+   return val * small + (1.0 - val) * big;
+}
+
+double ori_values(const Vector &x)
```

Comment:
Is this used anywhere?

Reply:
No... I think I was using it initially for orientation adaptivity, but not anymore. I can remove that function and push the changes.

Reply:
ok, go